AUTHORS

```diff
@@ -27,4 +27,5 @@ former developers:
 - Dirk Reith                (Fraunhofer Institute SCAI, Germany)
 - Victor Ruehle             (University of Cambridge, UK)
 - Vitalii Starchenko        (Max Planck Institute for Polymer Research, Germany)
-
+- Aoife Fogarty             (Max Planck Institute for Polymer Research, Germany)
+- Marc Radu                 (Max Planck Institute for Polymer Research, Germany)
```

Comment:
I think the lower listing is suppose to be alphabetical....

Reply:
You are right!